cirq/work/sampler.py

```diff
@@ -172,9 +170,7 @@ def run_sweep(
         Returns:
             An awaitable TrialResult.
         """
-        results = await self.run_sweep_async(program, study.UnitSweep,
-                                             repetitions)
-        return results[0]
+        return self.run(program, repetitions=repetitions)
```

Comment:
The comment

```
By default, this method calls `run` on another thread...
```

above needs to be updated.

(Applies below, too.)

Reply:
Done.